### ISSUE TITLE
Is this functioning?

### ISSUE THREAD

**Issue by avanibhatnagar** [rainermesi/tableau_xml_parse#1]
Hello,

Tried to use this - but I'm getting KeyErrors for all columns:

```
  File "xml_parse.py", line 28, in xml_parse
    col_names['role'].append(column.attrib['role'])
KeyError: 'role'
```

The above happens to all - role, datatype and name.

So I'm wondering if this is functioning code? I used the xml_parse_func.py file

**Comment by rainermesi** [comment]
Nope, this definitely needs more work. I got sidetracked. Not all elements that the script is looping through have (role, datatype, etc) XML tags. You can push through the KeyErros to get some of the columns back.
